Tests/Marketplace/search_and_install_packs.py

```diff
@@ -319,86 +336,71 @@ def install_packs(client: demisto_client,
             try:
-                logging.info(f'Installing packs {", ".join([p.get("id") for p in packs_to_install])} on server {host}. '
-                             f'Attempts left on failure: {attempts_count - attempt - 1}.')
-                response_data, status_code, _ = demisto_client.generic_request_func(client,
-                                                                                    path='/contentpacks/marketplace/install',
-                                                                                    method='POST',
-                                                                                    body={'packs': packs,
-                                                                                          'ignoreWarnings': True},
-                                                                                    accept='application/json',
-                                                                                    _request_timeout=request_timeout,
-                                                                                    response_type='object')
-
-                if status_code in range(200, 300) and status_code != 204:
-                    packs_data = [{'ID': pack.get('id'), 'CurrentVersion': pack.get('currentVersion')} for pack in response_data]
+                logging.info(f"Installing packs {', '.join([p.get('id') for p in packs_to_install])} on server {host}. "
+                             f"Attempt:{attempts_count - attempt}/{attempts_count}")
+                response, status_code, headers = demisto_client.generic_request_func(client,
+                                                                                     path='/contentpacks/marketplace/install',
+                                                                                     method='POST',
+                                                                                     body={'packs': packs_to_install,
+                                                                                           'ignoreWarnings': True},
+                                                                                     accept='application/json',
+                                                                                     _request_timeout=request_timeout,
+                                                                                     response_type='object')
+
+                if 200 <= status_code < 300 and status_code != 204:
+                    packs_data = [{'ID': pack.get('id'), 'CurrentVersion': pack.get('currentVersion')} for pack in response]
                     logging.success(f'Packs were successfully installed on server {host}')
                     logging.debug(f'The packs that were successfully installed on server {host}:\n{packs_data}')
                     break
 
+                if not attempt:
+                    raise Exception(f"Got bad status code: {status_code}, headers:{headers}")
```

Comment:
Can `headers` contain sensitive info?

Also:
```suggestion
                    raise Exception(f"Got bad status code: {status_code}, headers: {headers}")
```